code/script.js

```diff
@@ -29,23 +37,246 @@ const showMessage = (message, sender) => {
       </section>
     `
   }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
   chat.scrollTop = chat.scrollHeight;
 }
 
-// Starts here
-const greetUser = () => {
-  // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot');
-  // Just to check it out, change 'bot' to 'user' here 👆
+//Simplifies calling for user and bot reply in the chat bubbles, adding a delay for both types.
```

Comment:
Nice comment explaining why variables are declared. Also prevents some cluttering of following code.